src/terra/Factory/EnvironmentFactory.php

```diff
@@ -648,6 +666,39 @@ public function writeDrushAlias()
     }
 
     /**
+     * Writes a local Drupal Console site file.
+     */
+    public function writeConsoleSiteTargets()
+    {
+        $console_target_file_path = "{$_SERVER['HOME']}/.console/sites/terra.{$this->app->name}.yml";
+
+        $console_target_file = array();
+        $console_target_file[] = '';
+
+        foreach ($this->app->environments as $environment_name => $environment) {
+            $factory = new self($environment, $this->app);
+            $path = "/source/" . $environment['document_root'];
+            $console_target_file[] = '# DO NOT EDIT. This is generated by Terra. Any changes will be overridden when the environment is re-enabled.';
+            $console_target_file[] = "{$environment_name}:";
+            $console_target_file[] = "  host: {$factory->getHost()}";
+            $console_target_file[] = "  root: $path";
+            $console_target_file[] = "  user: root";
+            $console_target_file[] = "  port: {$factory->getDrupalConsolePort()}";
```

Comment:
This also means we don't need a separate `getDrupalConsolePort()`  if we can leverage `getDrushPort()`. 

This will simplify things a lot I believe.